pages/o-serwisie.tsx

```diff
@@ -0,0 +1,23 @@
+import Link from 'next/link';
+
+import { AboutNote } from '../components/AboutNote/AboutNote';
+import { Button } from '../components/Button/Button';
+import { Layout } from '../components/Layout';
+import { Section } from '../components/Section/Section';
+
+export default function AboutPage() {
+  return (
+    <Layout>
+      <Section
```

Comment:
Czym to się różni od komponentu Layout?

Reply:
Abstrakcja na sekcję z tytułem i przyciskami

Reply:
To nie odpowiada na moje pytanie :P